docs/getting-started/numpy-pandas-xarray.ipynb

```diff
@@ -0,0 +1,296 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "a2d = np.sin(0.15 * np.arange(50.0)).reshape(50, 1) * np.sin(0.2 * np.arange(30.0))\n",
+    "pp.plot(a2d)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "13fe5c3e-aeed-4ef8-9496-f7855df317c7",
+   "metadata": {},
+   "source": [
+    "Just like with Scipp data arrays, plotting multiple arrays onto the same axes is achieved by supplying a dict to the `plot` function:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3aafb43d-a0ff-4b06-80e1-7402f1d0dafc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "b1d = 3 * a1d + np.random.random(50)\n",
+    "pp.plot({'a': a1d, 'b': b1d})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "538eba83-7e7b-4e9e-a615-0c26c8ae768d",
+   "metadata": {},
+   "source": [
+    "## Pandas Series and DataFrame\n",
+    "\n",
+    "Plopp's `plot` wrapper will accept a Pandas data `Series` as input in the same way:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6f41ff1b-c660-446e-8b57-1fb1ca83966a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "N = 200\n",
+    "ts = pd.Series(\n",
+    "    np.random.randn(N), index=pd.date_range(\"1/1/2000\", periods=N), name='Temperature'\n",
+    ")\n",
+    "ts = ts.cumsum()\n",
+    "pp.plot(ts, ls='-', marker=None)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "659818d6-5d87-49fc-906b-8fdc26da2d32",
+   "metadata": {},
+   "source": [
+    "Supplying a `DataFrame` to `plot` will attempt to place all entries on the same axes.\n",
+    "This is very useful for quick inspection,\n",
+    "but it also means that if some data types are incompatible (e.g. some columns are floats, while others are strings),\n",
+    "the call to `plot` will fail."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a6e03a0f-94bf-49b6-ae5b-7dad36c01ddd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = pd.DataFrame(np.random.randn(N, 4), index=ts.index, columns=list(\"ABCD\"))\n",
+    "df = df.cumsum()\n",
+    "pp.plot(df, ls='-', marker=None)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "743fe2ac-2601-498d-8b26-9f3f733516d4",
+   "metadata": {},
+   "source": [
+    "## Xarray\n",
+    "\n",
+    "Xarray data structures are very similar to the ones Scipp provides,\n",
+    "and the labeled dimensions allow us to automatically annotate the axes labels of a figure."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ea400a36-0e1c-4329-a012-a3c04b2bdc54",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import xarray as xr\n",
+    "\n",
+    "air = xr.tutorial.open_dataset(\"air_temperature\").air\n",
+    "# We modify a few entries which are not well handled by Scipp\n",
+    "del air.attrs['precision']\n",
+    "del air.attrs['GRIB_id']\n",
+    "del air.attrs['actual_range']\n",
```

Comment:
Allowing numpy array in the attributes will help removing these lines but Simon suggested to postpone it until we decide whether to keep `attrs` or not.
We can maybe remove these lines after the changes. : (....

Reply:
Yes, I think for now we just keep it as is.